styles/main.scss

```diff
@@ -1,9 +1,44 @@
-$color: blue;
+#rajoyirize-container {
+    font-family: Arial;
+    font-size: 16px;
+    width: 100%;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    overflow: hidden;
+    
+    &.hidden {
+        display: none !important;
+    }
+    
+    .rajoyirize-content-wrapper {
```

Comment:
I'm sure you already tried that but... is this container really necessary? If you apply the same styling to the element `#rajoyirize-container`, which is already a container, is not enough?